models/message.py

```
@@ -0,0 +1,30 @@
+from .base import Base
+from sqlalchemy.dialects.postgresql import UUID
+import uuid
+
+class MessageType(str, PyEnum):
+    USER = "user"
+    AI = "ai"
+
+class Message(Base):
+    __tablename__ = "messages"
+    
+    from sqlalchemy.dialects.postgresql import UUID
```

Ruff (0.15.0)

[error] Redefinition of unused `UUID` from line 6: `UUID` redefined here

Remove definition: `UUID`

(F811)